app.py

```diff
@@ -310,6 +316,51 @@ def save_identity(caller_identity):
     return jsonify({})
 
 
+# End Point which creates payout info next to identity
+@app.route('/v1/identities/<identity_url_param>/payout', methods=['POST'])
+@validate_json
+@recover_identity
+def create_payout_info(identity_url_param, caller_identity):
+    payload = request.get_json(force=True)
+
+    identity_payload = payload.get('identity', None)
```

Comment:
Signature and identity recovery from signature was should be in header - same style as all other endpoints

Reply:
`create_payout_info` is the first and only one endpoint which needs to get signature from http headers. I would not suggest to provide signature parameter with others endpoints since they don't need them.